packages/@aws-cdk/aws-kinesisfirehose-destinations/package.json

```diff
@@ -77,7 +77,9 @@
     "cfn2ts": "0.0.0",
     "jest": "^26.6.3",
     "pkglint": "0.0.0",
-    "@aws-cdk/assert-internal": "0.0.0"
+    "@aws-cdk/assert-internal": "0.0.0",
+    "@aws-cdk/aws-kms": "0.0.0",
+    "@aws-cdk/aws-logs": "0.0.0"
```

Comment:
unused?

Reply:
Good catch, yep both KMS and logs can be removed.